networkx/drawing/tests/test_layout.py

```diff
@@ -371,6 +371,16 @@ def test_spiral_layout(self):
             distances_equidistant[1:], distances_equidistant[-1], atol=0.01
         )
 
+    @pytest.mark.mpl_image_compare
+    def test_paramter_equidistant_for_spiral_layout(self):
+        import matplotlib.pyplot as plt
+
+        fig = plt.figure()
+        G = nx.path_graph(5)
+        pos = nx.spiral_layout(G, equidistant=True)
+        nx.draw(G, pos=pos)
+        return fig
+
```

Comment:
This is a nice test! NetworkX has only recently adopted `pytest-mpl` and I think there is a goal to keep the number of saved images to an absolute minimum, at least until there's been a chance to think about how to organize the tests/image storage in an efficient way.

Fortunately, in this case it's relatively straightforward to design a test that evaluates the layout directly rather than an image. For example, how about a test that asserts that the distance between all the nodes is 0 (with a generous tolerance for all the floating-point computations):

```suggestion
    def test_spiral_layout_equidistant(self):
        G = nx.path_graph(10)
        pos = nx.spiral_layout(G, equidistant=True)
        # Extract individual node positions as an array
        p = np.array(list(pos.values()))
        # Elementwise-distance between node positions
        dist = np.linalg.norm(p[1:] - p[:-1], axis=1)
        assert np.allclose(np.diff(dist), 0, atol=1e-3)

```

This test should fail on `main`, pass on this branch, and has the advantage that it doesn't require any image storage!

Reply:
>> This is a nice test! NetworkX has only recently adopted pytest-mpl and I think there is a goal to keep the number of saved images to an absolute minimum, at least until there's been a chance to think about how to organize the tests/image storage in an efficient way.

Thanks for educating me on this one , will keep in mind for future , the approach you use is surely better ! Also rewording the docstring looks good . Overall looks good to go in !